examples/apps/cra-demo/README.md

```diff
@@ -72,18 +72,20 @@ Lastly, open up the `App.js` file, as that will be the only file we need to edit
 
 ## 3. <a style="position: relative; top: 20px" name="import"></a> Import and Initialize Fluid Dependencies
 
-`TinyliciousClient` is the service we will use to connect to our local Tinylicious server. It also provides methods to create a [Fluid container](https://fluidframework.com/docs/glossary/#container) with a set of initial [DataObjects](https://fluidframework.com/docs/glossary/#dataobject) or [DDSes](https://fluidframework.com/docs/concepts/dds/) that are defined in the `containerSchema`.
+`FrsClient` is a client that will allow us to connect connect to a local `Tinylicious` server while testing our application. It also provides methods to create a [Fluid container](https://fluidframework.com/docs/glossary/#container) with a set of initial [DataObjects](https://fluidframework.com/docs/glossary/#dataobject) or [DDSes](https://fluidframework.com/docs/concepts/dds/) that are defined in the `containerSchema`.
```

Comment:
```suggestion
`FrsClient` is a client that will allow us to connect connect to a local `Tinylicious` server while testing our application. It also provides methods to create a [Fluid container](https://fluidframework.com/docs/glossary/#container) with a set of initial [DDSes](https://fluidframework.com/docs/concepts/dds/) or [DataObjects](https://fluidframework.com/docs/glossary/#dataobject) that are defined in the `containerSchema`.
```

Leading with ddses since that's what they're using in this example.